### ISSUE TITLE
Scale bar does not correspond to the actual size

### ISSUE THREAD

**Issue by alexandrejaguar** [ppinard/matplotlib-scalebar#14]
Dear all,
first of all, thank you very much for this, scalebars are kinda mandatory in research and having this in python is really nice.
I am having some issues when using ScaleBar. The bar does not correspond to the actual size it would have; could you help me with that?
I'm using this code:

```
from matplotlib_scalebar.scalebar import ScaleBar
from skimage.io import imread

len_px = 0.00048223304582578836

image = imread('dur_grain1apatite01.tif', as_grey=True)

fig, ax = plt.subplots(figsize=(15, 10))
ax.imshow(testimg_apatite)
scalebar = ScaleBar(dx=len_px,
                    units='mm',
                    fixed_value=25,
                    fixed_units='um',
                    location=4,
                    box_alpha=0,
                    font_properties={'family' : 'monospace',
                                     'weight' : 'semibold',
                                     'size' : 20})
fig.gca().add_artist(scalebar)
plt.show()
```
This is the [example image](https://www.dropbox.com/s/ijw26a8octk7duy/dur_grain1apatite01.tif?dl=0).
Thank you very much for your support.

**Comment by ppinard** [comment]
Thank you both for your comment. You are 100% right there was a mistake in the code. After many attempts to understand the problem I realized that the "bar" of the scale bar was drawn with an edge around it :disappointed: This made the "bar" appeared longer. Making the bar longer (with the ``length_fraction``) helped because the error due to the edge decreases as the length of the bar increases.

I tried @parishcm's example and I got a length of 10.0769 in ImageJ. I committed a fix and I will release a new version soon.
